src/NUnitFramework/framework/Assert.cs

```diff
@@ -374,7 +374,7 @@ private static void IssueWarning(string message)
             result.RecordAssertion(AssertionStatus.Warning, message, GetStackTrace());
         }
 
-        // System.Envionment.StackTrace puts extra entries on top of the stack, at least in some environments
+        // System.Environment.StackTrace puts extra entries on top of the stack, at least in some environments
```

Comment:
I'd really like to use `<see cref="System.Environment.StackTrace"/>` here but that should probably be its own PR and take care of more than just the typo spots.